charts/dapr/README.md

```diff
@@ -76,7 +76,7 @@ The Helm chart has the follow configuration options that can be supplied:
 | Parameter                                 | Description                                                             | Default                 |
 |-------------------------------------------|-------------------------------------------------------------------------|-------------------------|
 | `global.registry`                         | Docker image registry                                                   | `docker.io/daprio`      |
-| `global.tag`                              | Docker image version tag                                                | `1.6.0`            |
+| `global.tag`                              | Docker image version tag                                                | latest release          |
```

Comment:
Why not this also as DAPR_VERSION ?

Reply:
Because this is consumed directly from the repo and now from the helm chart.